rum-demo-app-ios/RUM Mobile Demo App/Extensions/UIViewControllerExtension.swift

```diff
@@ -0,0 +1,588 @@
+        button.frame = CGRect(x: 0.0, y: 0.0, width: 30.0, height: 30.0)
+        button.addTarget(self, action: #selector(gotoPreviousScreen), for: .touchUpInside)
+        
+        
+        let barButtonItem = UIBarButtonItem(customView: button)
+        
+        self.navigationItem.leftBarButtonItem = barButtonItem
+        
+        let button1 = UIButton(type: .custom)
+        button1.setImage(UIImage (named: "chevron-left"), for: .normal)
+        button1.frame = CGRect(x: 0.0, y: 0.0, width: 30.0, height: 30.0)
+        button1.addTarget(self, action: #selector(gotoPreviousScreen), for: .touchUpInside)
+        
+        
+        let barButtonItem1 = UIBarButtonItem(customView: button1)
+        
+        self.navigationItem.rightBarButtonItem = barButtonItem1
+    }
+    
+    
+    @objc private func handlePopViewController() {
+        self.navigationController?.popViewController(animated: true)
+    }
+    
+    @objc private func handleDismissVCWithoutNotification() {
+        self.dismiss(animated: true) {
+            return
+        }
+    }
+    
+    @objc func handleDismissKeyboard() {
+        for textField in self.view.subviews where textField is UITextField {
+            textField.resignFirstResponder()
+        }
+    }
+    
+    @objc func handleDismissSettings() {
+        self.dismiss(animated: false, completion: nil)
+    }
+    
+    // MARK: - nav bar button action
+    @objc func gotoURLConfigScreen() {
+        
+        NavMenu.showNavMenu {
+            let loginNavController = mainStoryBoard.instantiateViewController(withIdentifier: "LoginNavigationController")
+            (UIApplication.shared.delegate as? AppDelegate)?.changeRootViewController(loginNavController)
+        }
+    }
+    @objc func gotoPreviousScreen() {
+        navigationController?.popViewController(animated: true)
+    }
+    
+    func showMenuOption() {
+        
+        DispatchQueue.main.async {
+            let viewMainContainer = UIView.init(frame: window?.bounds ?? .zero)
+            viewMainContainer.backgroundColor = UIColor.black.withAlphaComponent(0.3)
+            let viewForMenu = UIView()
+            let lblTitle = UILabel.init(frame: CGRect.init(x: 0, y: 0, width: 150, height: 45))
+            lblTitle.text = "Change Config URL"
+            lblTitle.textAlignment = .center
+            lblTitle.textColor = .black
+            lblTitle.font = UIFont.systemFont(ofSize: 14)
+            lblTitle.backgroundColor = .white
+            viewForMenu.addSubview(lblTitle)
+            viewForMenu.frame.origin.x = UIScreen.main.bounds.width - (lblTitle.frame.width + 20)
+            viewForMenu.frame.origin.y = (window?.safeAreaInsets.top ?? 0) + 15
+            viewForMenu.frame.size.height = 45
+            viewForMenu.backgroundColor = .white
+            
+            
+            
+    //        let tapGesture = UITapGestureRecognizer.init(target: self, action: #selector(changeConfigURLButtonClicked(_:)))
```

Comment:
@akashCrest Can we remove this commented block of code?

Reply:
Done